### PR TITLE
Item: Update to v1 API

### PR DESCRIPTION
**Pull request by layershifter**
# Finished

Update [Item](http://semantic-ui.com/views/item.html) component to v1 API.

### CODE REVIEW

**Comment by layershifter** [comment]
@levithomason `Item.Items` was removed, what I must do with [this](https://github.com/TechnologyAdvice/stardust/blob/master/src/deprecatedComponents.js#L45)?

File: src/views/Item/ItemImage.js [hunk 1]
```diff
@@ -0,0 +1,15 @@
+import React, { PropTypes } from 'react'
+import { META } from '../../lib'
+import Image from '../../elements/Image/Image'
+
+function ItemImage(props) {
+  return <Image {...props} />
```
Comment (layershifter) [inline src/views/Item/ItemImage.js]: @levithomason I have problem there.
1. I think I need use `createImage` factory, but will it pass props?
2. `ItemImage` doesn't have `ui` class by default. How we can deal with this?

**Comment by layershifter** [comment]
@levithomason, finished :v: 

Can be merged after [issue will be solved](https://github.com/TechnologyAdvice/stardust/pull/431#discussion_r76516759) and code review :smiley_cat:

**Comment by levithomason** [comment]
Taking a look...

**Comment by levithomason** [comment]
I've added a `ui` prop (and tests) to Image to allow turning off the `ui` className:

``` jsx
<Image {...rest} as={ElementType} ui={false} />
```

This seems to solve the issue with the `ui` className simply.  We may add this feature for all components, though for now it will just be on the Image.

**Comment by levithomason** [comment]
There are a couple warnings to solve, looks like we're spreading props we shouldn't be:

```
Warning: Failed prop type: Invalid prop `items[0].childKey` of type `number` supplied to `ItemGroup`, expected `string`.
```

```
Warning: Unknown prop `childKey` on <div> tag. Remove this prop from the element.
in ItemGroup
```

**Comment by layershifter** [comment]
Seems, it fixed :+1:

File: src/elements/Image/Image.js [hunk 2]
```diff
@@ -20,11 +20,11 @@ import ImageGroup from './ImageGroup'
 function Image(props) {
   const {
     verticalAlign, alt, avatar, bordered, centered, className, disabled, floated, fluid,
-    hidden, height, href, inline, shape, size, spaced, src, width,
+    hidden, height, href, inline, shape, size, spaced, src, width, ui,
   } = props
 
   const classes = cx(
-    'ui',
+    ui && 'ui',
```
Comment (layershifter) [inline src/elements/Image/Image.js]: Stupid question. Why you used there '&&' instead of `useKeyOnly`? :rabbit:

Comment (levithomason) [reply src/elements/Image/Image.js]: Stupid answer, whoops!

**Comment by levithomason** [comment]
Will merge/release once tests pass

**Comment by levithomason** [comment]
Released in `v0.38.0`, we're getting closer to v1 :D 🎉

**Pull request merged**